codecov.yml

```diff
@@ -0,0 +1,46 @@
+# Enable coverage report message for diff on commit
```

Comment:
Now I remember that it's not a hidden file. It's one of the things I didn't like ;-)